src/plugins/intel_gpu/tests/test_cases/scatter_nd_update_gpu_test.cpp

```diff
@@ -12,17 +14,1154 @@
+                               format::b_fs_yx_fsv16, format::b_fs_yx_fsv16, format::b_fs_yx_fsv16,
+                               {48, 24, 3, 3}, {3, 2, 1, 1}, {3, 3, 1, 3},
+                               2 } 
+                         }));
+
+INSTANTIATE_TEST_SUITE_P(scatter_nd_update_gpu_random_test_bsv32_fsv16_i8, 
+                         scatter_nd_update_random_test,
+                         testing::ValuesIn( 
+                             std::vector<scatter_nd_update_basic_test_params>{
+                             { data_types::i8, data_types::i8, data_types::i8, 
+                               format::bfyx, format::bfyx, format::bfyx,
+                               format::bs_fs_yx_bsv32_fsv16, format::bs_fs_yx_bsv32_fsv16, format::bs_fs_yx_bsv32_fsv16,
+                               {41, 23, 3, 3}, {3, 2, 1, 1}, {3, 3, 1, 3},
+                               2 } 
+                         }));
+
+INSTANTIATE_TEST_SUITE_P(scatter_nd_update_gpu_random_test_bsv32_fsv32_i8, 
+                         scatter_nd_update_random_test,
+                         testing::ValuesIn( 
+                             std::vector<scatter_nd_update_basic_test_params>{
+                             { data_types::i8, data_types::i8, data_types::i8, 
+                               format::bfyx, format::bfyx, format::bfyx,
+                               format::bs_fs_yx_bsv32_fsv32, format::bs_fs_yx_bsv32_fsv32, format::bs_fs_yx_bsv32_fsv32,
+                               {41, 23, 3, 3}, {3, 2, 1, 1}, {3, 3, 1, 3},
+                               2 } 
+                         }));
+
+INSTANTIATE_TEST_SUITE_P(scatter_nd_update_gpu_random_test_fsv32_i8, 
+                         scatter_nd_update_random_test,
+                         testing::ValuesIn( 
+                             std::vector<scatter_nd_update_basic_test_params>{
+                             { data_types::i8, data_types::i8, data_types::i8, 
+                               format::bfyx, format::bfyx, format::bfyx,
+                               format::b_fs_yx_fsv32, format::b_fs_yx_fsv32, format::b_fs_yx_fsv32,
+                               {41, 23, 3, 3}, {3, 2, 1, 1}, {3, 3, 1, 3},
+                               2 } 
+                         }));
+
+INSTANTIATE_TEST_SUITE_P(scatter_nd_update_gpu_random_test_fsv16_i8, 
+                         scatter_nd_update_random_test,
+                         testing::ValuesIn( 
+                             std::vector<scatter_nd_update_basic_test_params>{
+                             { data_types::i8, data_types::i8, data_types::i8, 
+                               format::bfyx, format::bfyx, format::bfyx,
+                               format::b_fs_yx_fsv16, format::b_fs_yx_fsv16, format::b_fs_yx_fsv16,
+                               {41, 23, 3, 3}, {3, 2, 1, 1}, {3, 3, 1, 3},
+                               2 } 
+                         }));
+
+TEST(scatter_nd_update_gpu_fp16_fsv16_test14, data5_indice2_update3) {
```

Comment:
why cant these tests be merged to the parameterized test above ?

Reply:
unnecessary test cases have been merged